app/qml/components/MMIcon.qml

```diff
@@ -23,7 +24,8 @@ Item {
     id: icon
 
     source: __style.arrowLinkRightIcon
-    anchors.centerIn: control
+    anchors.fill: root.useCustomSize ? root : undefined
+    anchors.centerIn: root.useCustomSize ? undefined : root
```

Comment:
What is the point here? Let's directly create `Image` instead when you need something special :)

Reply:
https://doc.qt.io/qt-6/qml-qtquick-image.html#fillMode-prop